paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -463,16 +464,14 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 
         conv_pd = ConvFwdPrimitiveDesc(
             src_md, weights_md, bias_md, dst_md, strides, paddings,
-            mkldnn_engine, fuse_relu, fuse_residual_conn, false /*fuse_brelu*/,
-            0.0 /*fuse_brelu_threshold*/, output_shift_scale, sum_scale,
-            is_test);
+            mkldnn_engine, fuse_relu || fuse_brelu, fuse_residual_conn, false,
+            0.0, output_shift_scale, sum_scale, is_test);
 
       } else {
-        conv_pd = ConvFwdPrimitiveDesc(src_md, weights_md, dst_md, strides,
-                                       paddings, mkldnn_engine, fuse_relu,
-                                       fuse_residual_conn, false /*fuse_brelu*/,
-                                       0.0 /*fuse_brelu_threshold*/,
-                                       output_shift_scale, sum_scale, is_test);
+        conv_pd = ConvFwdPrimitiveDesc(
+            src_md, weights_md, dst_md, strides, paddings, mkldnn_engine,
+            fuse_relu || fuse_brelu, fuse_residual_conn, false, 0.0,
```

Comment:
`false, 0.0`, please see 'It will be removed once the int8 is enabled.' https://github.com/PaddlePaddle/Paddle/pull/17130#discussion_r285882103

Reply:
> `false, 0.0`, please see 'It will be removed once the int8 is enabled.' [#17130 (comment)](https://github.com/PaddlePaddle/Paddle/pull/17130#discussion_r285882103)

Hi, luotao, I changed this.

Reply:
I  just found CreatePostOps made problems. As in https://github.com/lidanqing-intel/Paddle/commits/develop-mobilnetv2, only last commit makes the accuracy 0. All previous commits are giving good accuracy. I will fix during the day, also will talk with @wojtuss.